iam-firewall-ips/iam-firewall-ips-sao.txt

```diff
@@ -1,11 +1,13 @@
 163.107.65.32/28
+163.107.65.70
```

Comment:
Same question as above (and there are some more IPs below) regarding some IPs now not having a /XX suffix